CMakeLists.txt

```diff
@@ -379,6 +401,13 @@ foreach(target devilution devilutionx)
     target_link_libraries(${target} PUBLIC -fsanitize=undefined)
   endif()
 
+  if(SWITCH)

```

Comment:
@glebm are you able to spot what is causing the build to fail for the switch target?

```
CMake Error at CMakeLists.txt:388 (target_link_libraries):

  Cannot specify link libraries for target "devilutionx" which is not built

  by this project.

```

Reply:
`foreach(target devilution devilutionx)` should be changed to `foreach(target devilution ${BIN_TARGET})`